miniflux.1

```diff
@@ -247,6 +247,10 @@ Default is 20 seconds\&.
 Maximum body size for HTTP requests in Mebibyte (MiB)\&.
 .br
 Default is 15 MiB\&.
+.B HTTP_CLIENT_PROXY
+Proxy url for HTTP client fetch feed content\&.
+.br
+Default is empty \&.
```

Comment:
There is an extra space before `\&`. The wording above can be only `Proxy URL for HTTP client`.